src/pages/module-builder/Document.tsx

```diff
@@ -66,29 +73,56 @@ const Document = ({
-                                        columns={section.properties.columns}
-                                    />
+                                    <Container
+                                        paddingTop={section.padding.top}
+                                        paddingRight={section.padding.right}
+                                        paddingBottom={section.padding.bottom}
+                                        paddingLeft={section.padding.left}
+                                    >
+                                        <MultipleChoicePreview
+                                            question={
+                                                section.multipleChoice.question
+                                            }
+                                            options={
+                                                section.multipleChoice.options
+                                            }
+                                            variant={modulePreviewVariant}
+                                            columns={section.properties.columns}
+                                        />
+                                    </Container>
                                 );
                             } else if (section.type === "multiSelect") {
                                 sectionPreview = (
-                                    <MultiSelectPreview
-                                        question={section.multiSelect.question}
-                                        options={section.multiSelect.options}
-                                        variant={modulePreviewVariant}
-                                        columns={section.properties.columns}
-                                    />
+                                    <Container
+                                        paddingTop={section.padding.top}
+                                        paddingRight={section.padding.right}
+                                        paddingBottom={section.padding.bottom}
+                                        paddingLeft={section.padding.left}
+                                    >
+                                        <MultiSelectPreview
+                                            question={
+                                                section.multiSelect.question
+                                            }
+                                            options={
+                                                section.multiSelect.options
+                                            }
+                                            variant={modulePreviewVariant}
+                                            columns={section.properties.columns}
+                                        />
+                                    </Container>
                                 );
                             }
                             return sectionPreview;
```

Comment:
You can make this code more DRY by wrapping the container here, rather than in each else if